library/src/main/java/ai/promoted/telemetry/DefaultTelemetry.kt

```diff
@@ -0,0 +1,83 @@
+package ai.promoted.telemetry
+
+import android.annotation.SuppressLint
+import android.content.Context
+import com.google.firebase.analytics.FirebaseAnalytics
+import com.google.firebase.analytics.ktx.logEvent
+import com.google.firebase.crashlytics.FirebaseCrashlytics
+
+private const val EVENT_METRICS_SENT = "metrics-sent"
```

Comment:
Hmm, does it make sense to use the same buckets as iOS? On iOS:
- event name for number of events sent is called `ai_promoted_event_count`, param `value` is the numeric value
- event name for number of bytes sent is called `ai_promoted_bytes_sent`, param `value` is the numeric value
- event name for errors is `ai_promoted_error`, param `description` is the error string. (Param `code` is error code and param `domain` is error domain, but I don't know if those concepts have analogues on Android.)

At the least, I believe that Firebase doesn't allow dashes in event names.

Reply:
yes let's sync about this today. I'm having trouble conceptualizing how it could be meaningful when articulated as "event count", since we'd only be logging it after a single batch was sent (as opposed to keeping a running tally for the lifetime of the app/user). Same thought for bytes sent.